packages/solarwinds-apm/src/init.ts

```diff
@@ -281,11 +281,14 @@ async function spanProcessors(
   if (config.dev.otlpTraces) {
     const { SwOtlpExporter } = await import("@solarwinds-apm/sdk/otlp-exporter")
     const exporter = new SwOtlpExporter(config)
+
+    const responseTimeProcessor = new sdk.SwResponseTimeProcessor(config)
```

Comment:
Currently the metric is generated any time spans are exported through OTel, would we want to make that condition different ? I'm not sure

Reply:
Woops my bad on the wording, it'll be generated for any recorded span. What I meant to say is that it is turned on any time OTLP is used.